packages/server/assets/gatekeeper/typedefs/gatekeeper.graphql

```diff
@@ -1,7 +1,3 @@
-extend type Query {
-  workspacePricingPlans: JSONObject!
```

Comment:
according to Apollo Studio hasn't been used by anyone since Jan 14th. the FE def doesn't use it